lib/product-import.coffee

```diff
@@ -230,6 +232,52 @@ class ProductImport
       else
         false
 
+  _updateProductRepeater: (prodToProcess, existingProduct) ->
+    repeater = new Repeater {attempts: 5}
```

Comment:
I was thinking 2, so if it fails more than that, we would know something else is wrong. 
And also, would be nicer if the unit test(s) were/was in the spec file (may cause duplication, but....)